packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -347,6 +355,16 @@ const NewNetwork = decorate([
                           ]}
                           value={encapsulation}
                         />
+                        <Toggle
+                          onChange={effects.toggleEncrypted}
+                          value={encrypted}
+                        />{' '}
+                        <label>{_('newNetworkEncrypted')}</label>
```

Comment:
I like for the label to be clickable and linked to the form input, but I don't think there is way to do this with our `Toggle` component :confused: